src/test/java/org/springframework/data/elasticsearch/core/ElasticsearchIntegrationTests.java

```diff
@@ -1872,6 +1874,7 @@ public void shouldReturnScores() {
 				IndexCoordinates.of(indexNameProvider.indexName()));
 
 		assertThat(searchHits.getMaxScore()).isGreaterThan(0f);
+		assertThat(searchHits.getExecutionDuration().toMillis()).isGreaterThan(0);
```

Comment:
Please add "#2986" to the comment in line 1861 (`@Test // DATAES-462`)

Reply:
Done.